src/random_ray/flat_source_domain.cpp

```diff
@@ -158,31 +158,31 @@ void FlatSourceDomain::update_neutron_source(double k_eff)
     }
   }
 
-  if (settings::run_mode == RunMode::EIGENVALUE) {
```

Comment:
So it looks like Flat Source random ray wasn't pulling fission cross sections _at all_ for eigenvalue calculations? Great catch, glad it's a simple fix!

Reply:
For eigenvalue calculations it was including the fission cross sections as expected, but for fixed source w/flat sources, it was indeed omitting any fission contributions.

Reply:
Ah, I miswrote, yeah that's what I meant to say.